src/repositories/news.repository.ts

```diff
@@ -0,0 +1,12 @@
+import { Repository } from '../core/repository';
+import { NewsRepositoryInboxResponseRootObject } from '../responses';
+
+export class NewsRepository extends Repository {
+  public async inbox(): Promise<NewsRepositoryInboxResponseRootObject> {
```

Comment:
Isn't that a feed?

Reply:
I thought the same, but on none of my accounts this list is scrollable (no additional request) even on such where the last item is 10min ago.